CanDIG/katsu
# add schema to static folder

## What's new
- Add schemas folder into staticfiles
- Move schemas in docs into its own folder
- Modify path script

### How to test
- Rebuild katsu with this branch
- Visit `http://candig.docker.internal:8008/static/schema.json` to confirm the schema
